examples/jsm/loaders/LogLuvLoader.js

```diff
@@ -55,7 +55,7 @@ UTIF.decode = function ( buff, prm ) {
 	var bin = id == 'II' ? UTIF._binLE : UTIF._binBE;
 	bin.readUshort( data, offset ); offset += 2;
 
-	var ifdo = bin.readUint( data, offset ); offset += 4;
+	var ifdo = bin.readUint( data, offset );
```

Comment:
This one seems suspicious... 🤔

Reply:
It's not necessary to update `offset` at this point since there is no access to the variable anymore.

Reply:
Okay! 👍